Cargo.toml

```diff
@@ -55,6 +54,9 @@ toml = "0.4.2"
 url = "1.1"
 clap = "2.31.2"
 unicode-width = "0.1.5"
+libssh2-sys = "0.2.8"
+libgit2-sys = "0.7.5"
+gcc = "0.3.23"
```

Comment:
Could a comment be added saying that some of these deps are synthetic? Also I think `gcc` should be in `[build-dependencies]`, right?

Reply:
good thought. I added comments.